### PR TITLE
Updated prospect script with the correct URLs

### PR DESCRIPTION
**Pull request by dominicaw**
Updated the prospect form script to point to the production farsight API `/farsight/api/v1/company/prospects ` and updated the page redirect to to go `/hire-developers/get-started`

### CODE REVIEW

File: offerzen/global/company-fast-track-access/company-prospect-form@v1.1.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        }
+      }
+
+      function startProspectPolling(prospectId, buttonLabelTimer) {
+        const errorText = formContainer.find('.js-form-error');
+
+        function poll() {
+          $.ajax({
+            type: 'GET',
+            url: `/farsight/api/v1/company/prospects/${prospectId}`,
```
Comment (dominicaw) [inline offerzen/global/company-fast-track-access/company-prospect-form@v1.1.js:72]: farsight production api here

File: offerzen/global/company-fast-track-access/company-prospect-form@v1.1.js [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+              Accept: 'application/json',
+            },
+            statusCode: {
+              304: function () {
+                setTimeout(poll, 1000);
+              },
+              200: function (data) {
+                clearTimeout(buttonLabelTimer);
+                trackSubmission();
+                window.location.href = `/hire-developers/get-started?prospect_id=${prospectId}`;
```
Comment (dominicaw) [inline offerzen/global/company-fast-track-access/company-prospect-form@v1.1.js:84]: redirect URL here

File: offerzen/global/company-fast-track-access/company-prospect-form@v1.1.js [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+          // Hide errors
+          form.find('.js-missing-fields').hide();
+          const prospectProperties = Object.assign({}, formProperties, {
+            referrer: location.href,
+            report_source: searchParams.get('report_source'),
+            page_variant_meta: `${window.pageVariantMeasureStart}-optimize-meta-${pageVariantMeasureEnd}`,
+          });
+          $.ajax({
+            type: 'POST',
+            url: '/farsight/api/v1/company/prospects/',
```
Comment (dominicaw) [inline offerzen/global/company-fast-track-access/company-prospect-form@v1.1.js:142]: farsight API url here

**Review by jethroflanagan — APPROVED** [review]
Thanks for marking the changes!

**Review by MarkusKuhn — APPROVED** [review]

**Pull request merged**